app/scripts/contentscript.js

```diff
@@ -37,8 +37,10 @@ function injectScript (content) {
   try {
     const container = document.head || document.documentElement
     const scriptTag = document.createElement('script')
+    scriptTag.setAttribute('async', false)
     scriptTag.textContent = content
     container.insertBefore(scriptTag, container.children[0])
+    container.removeChild(scriptTag)
```

Comment:
Should we remove this after the script has loaded?

Reply:
@whymarrh it was my understanding that dynamically-added script tags with `async=false` will be executed synchronously. I verified this crudely using the following code in the console:

```js
const container = document.head || document.documentElement
const scriptTag = document.createElement('script')
scriptTag.setAttribute('async', false)
scriptTag.textContent = 'for(var i = 0; i< 10000; i++) { console.log(i); }'
container.insertBefore(scriptTag, container.children[0])
console.log('done')
```

Reply:
Yup, that makes sense (script tags are synchronous, I forgot that)